source/stylesheets/main.css.scss

```diff
@@ -2,7 +2,34 @@
 
 // App-specific components
 @import "components/header";
+@import "components/divider";
+@import "components/masthead";
 
 body {
   margin: 0;
 }
+
+// Mirrors Bootstrap 4 - open for discussion
+$mq-breakpoint-widescreen: 1200px;
+
+// This will be coming to FE later but making it app specific for now
+.app-c-site-width-container {
+  @media (min-width: $mq-breakpoint-widescreen) {
+    max-width: 1100px;
+	}
```

Comment:
Think you've got a mix of tabs and spaces in this file for indentation.

Reply:
This should be fixed now.